pyproject.toml

```diff
@@ -54,3 +54,6 @@ known_first_party = ["django_filters"]
 
 [tool.flit.module]
 name = "django_filters"
+
+[tool.coverage.run]
+relative_files = true
```

Comment:
Because this was added to `pyproject.toml`, I had to add the extra `[toml]` in the placeS where `coverage is installed. It wouldn't be the case if we put the `coverage` config in a different file, but it feels right to put it here. Feel free to indicate if you'd like it another way.